### PR TITLE
Issue 823 improve textcha for registration

### PR DESCRIPTION
**Pull request by ethancarlsson**
#823 

Textchas will now be created using regular expressions to allow multiple possible answers. Ignores case for all answers. Model now includes a verbose name describing the need to use regular expressions. Altered the test slightly to better test the new functionality.

### CODE REVIEW

File: conference/models.py [hunk 1]
```diff
@@ -1294,7 +1294,12 @@ class NoQuestionsAvailable(Exception):
         pass
 
     question = models.CharField(max_length=255)
-    answer   = models.CharField(max_length=255)
+    answer   = models.CharField(max_length=255,
+                                verbose_name='''answer (use a regular expression
+                                to capture possible answers e.g.
+                                "(python|the python programming language)"
+                                case is ignored)
+                                ''')
```
Comment (malemburg) [inline conference/models.py:1302]: Doesn't this change require a migration as well ?

**Review by malemburg — COMMENTED** [review]
Other than the minor nit about migrations, this looks good.

**Comment by malemburg** [comment]
Hi @ethancarlsson, there's a problem with the PR: it seems you locally switched to ep2021, but the PR still lists dev/ep2021 as target. As a result, the PR tries to merge all the ep2021 changes back into dev/ep2021. Could you fix that please ? Thanks.

**Comment by malemburg** [comment]
It may be easiest to close this PR and simply open a new one targeting the right branch.

**Comment by ethancarlsson** [comment]
Hi, yeah sorry better to close this one and start again. I didn't notice I had done that.

**Pull request closed without merging**